src/commands/run.rs

```diff
@@ -42,38 +43,211 @@ Configuration is loaded from the following sources (highest precedence first):
+    #[structopt(
+        long = "--log-destination",
+        env = "LOG_DESTINATION",
+        default_value = "stdout",
+        verbatim_doc_comment
+    )]
+    pub log_destination: tracing::LogDestination,
+
+    /// Tracing: exporter type
+    ///
+    /// Can be one of: none, jaeger, otlp
+    ///
+    /// When enabled, additional flags are considered (see flags related to OTLP
+    /// and Jaeger), and log output is disabled.
+    ///
+    /// TODO(jacobmarble): allow logs and traces simultaneously
+    #[structopt(
+        long = "--traces-exporter",
+        env = "TRACES_EXPORTER",
+        default_value = "none"
+    )]
+    pub traces_exporter: tracing::TracesExporter,
+
+    /// Tracing: filter directive
+    ///
+    /// Configures traces severity level filter, by target.
+    ///
+    /// Simplest options: error, warn, info, debug, trace
+    ///
+    /// Levels for different modules can be specified. For example
+    /// `debug,hyper::proto::h1=info` specifies debug logging for all modules
+    /// except for the `hyper::proto::h1' module which will only display info
+    /// level logging.
+    ///
+    /// Extended syntax provided by `tracing-subscriber` includes span/field
+    /// filters. See the link below for more details.
+    ///
+    /// No filter by default.
+    ///
+    /// https://docs.rs/tracing-subscriber/0.2.17/tracing_subscriber/filter/struct.EnvFilter.html
+    #[structopt(long = "--traces-filter", env = "TRACES_FILTER")]
+    pub traces_filter: Option<String>,
+
+    /// Tracing: sampler type
+    ///
+    /// Can be one of:
+    /// always_on, always_off, traceidratio,
+    /// parentbased_always_on, parentbased_always_off, parentbased_traceidratio
+    ///
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.1.0/specification/sdk-environment-variables.md#general-sdk-configuration
+    #[structopt(
+        long = "--traces-sampler",
+        env = "TRACES_SAMPLER",
+        default_value = "parentbased_traceidratio"
+    )]
+    pub traces_sampler: tracing::TracesSampler,
+
+    #[rustfmt::skip]
+    /// Tracing: sampler argument
+    ///
+    /// Valid range: [0.0, 1.0].
+    /// 
+    /// Only used if OTEL_TRACES_SAMPLER is set to
+    /// parentbased_traceidratio (default) or traceidratio.
+    ///
+    /// With sample parentbased_traceidratio, the following rules apply:
+    /// - if parent is sampled, then all of its children are sampled
+    /// - else sample this portion of traces (0.5 = 50%)
+    ///
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/v1.1.0/specification/sdk-environment-variables.md#general-sdk-configuration
+    #[structopt(
+        long = "--traces-sampler-arg",
+        env = "TRACES_SAMPLER_ARG",
+        default_value = "1.0",
+        verbatim_doc_comment
+    )]
+    pub traces_sampler_arg: f64,
+
+    /// Tracing: OTLP (eg OpenTelemetry collector) network hostname
+    ///
+    /// Only used if TRACES_EXPORTER is "otlp".
```

Comment:
Same comment here, should this be:

```suggestion
    /// Only used if `--traces-exporter` is "otlp".
```